content/ko/docs/contribute/new-content/overview.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 새로운 콘텐츠 기여하기에 대한 개요
+linktitle: 개요
+content_template: templates/concept
+main_menu: true
+weight: 5
+---
+
+{{% capture overview %}}
+
+이 섹션에는 새로운 콘텐츠를 기여하기 전에 알아야 할 정보가 있다.
+
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## 기여하기에 대한 기본
+
+- 마크다운(markdown)으로 쿠버네티스 문서를 작성하고 [Hugo](https://gohugo.io/)를 사용하여 쿠버네티스 사이트를 구축한다.
```

Comment:
nit (OK to merge without fixing) / might not be a bug:
```suggestion
- 마크다운(Markdown)으로 쿠버네티스 문서를 작성하고 [Hugo](https://gohugo.io/)를 사용하여 쿠버네티스 사이트를 구축한다.
```

Reply:
Miss-typed. :-)
I'll change it.